src/gameobjects/GameObject.js

```diff
@@ -148,7 +148,7 @@ var GameObject = new Class({
          * If this Game Object is enabled for physics then this property will contain a reference to a Physics Body.
          *
          * @name Phaser.GameObjects.GameObject#body
-         * @type {?object}
+         * @type {?object|Phaser.Physics.Arcade.Body|Phaser.Physics.Impact.Body}
```

Comment:
Pretty sure it needs to be {?(object|Phaser.Physics.Arcade.Body|Phaser.Physics.Impact.Body)}`

Reply:
Fixed.